elementary/monitor/alerts/source_freshness.py

```diff
@@ -31,17 +37,39 @@ def __init__(
     ) -> None:
         super().__init__(**kwargs)
         self.model_unique_id = model_unique_id
-        self.snapshotted_at = (
+        self.snapshotted_at_str = (
             convert_datetime_utc_str_to_timezone_str(snapshotted_at, self.timezone)
             if snapshotted_at
             else None
         )
+
         self.max_loaded_at = (
-            convert_datetime_utc_str_to_timezone_str(max_loaded_at, self.timezone)
+            convert_datetime_utc_str_to_timezone_datatime(max_loaded_at, self.timezone)
             if max_loaded_at
             else None
         )
+        self.max_loaded_at_str = (
+            self.max_loaded_at.strftime(DATETIME_FORMAT) if self.max_loaded_at else None
```

Comment:
Lets extend the util `convert_datetime_utc_str_to_timezone_str` to support `with_time_zone` which by default is false to maintain backwards, instead of using the strftime all the time.

If `with_time_zone` is false - use `DATETIME_FORMAT`
else - use `DATETIME_WITH_TIMEZONE_FORMAT`